INSTALL.md

```diff
@@ -1,46 +1,116 @@
-# 1. Back-end setup
+# Installation and Configuration for cfgov-refresh
 
-Follow the [Sheer installation instructions](https://github.com/cfpb/sheer#installation)
-to get Sheer installed.
-**NOTE:** We suggest creating a virtualenv variable specific to this project,
-such as `cfgov-refresh` instead of `sheer` used in the Sheer installation instructions:
+## 1. Back-end setup
 
+### Virtualenv & Virtualenvwrapper Python modules
+
+Install [Virtualenv](https://virtualenv.pypa.io/en/latest/index.html) and
+[virtualenvwrapper](https://virtualenvwrapper.readthedocs.org/en/latest/)
+to create a local environment for your server:
 ```bash
-$ mkvirtualenv cfgov-refresh
+pip install virtualenv virtualenvwrapper
 ```
 
-Install the following dependencies into your virtual environment.
-We called ours `cfgov-refresh` (see previous step above):
+### Autoenv module
+
+[Install Autoenv](https://github.com/kennethreitz/autoenv#install) however you'd like.
+(We use [Homebrew](http://brew.sh)):
 
+Run:
 ```bash
-$ workon cfgov-refresh
+brew install autoenv
+```
 
-$ pip install git+git://github.com/dpford/flask-govdelivery
-$ pip install git+git://github.com/rosskarchner/govdelivery
+After installation, Homebrew will output instructions similar to:
+```bash
+To finish the installation, source activate.sh in your shell:
+  source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh
 ```
 
-## Back-end environment variables
+Any time you run the project you'll need to run that last line. If you'll be working with
+the project consistently, we suggest adding it to your bash profile by running:
+```bash
+echo 'source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
+```
 
-The project needs a number of environment variables.
-The project uses a WordPress API URL to pull in content
-and GovDelivery for running the subscription forms:
+If you need to find this info again later, you can run:
+```bash
+brew info autoenv
+```
 
-- `WORDPRESS` (URL to WordPress install)
-- `GOVDELIVERY_BASE_URL`
-- `GOVDELIVERY_ACCOUNT_CODE` (GovDelivery account variable)
-- `GOVDELIVERY_USER` (GovDelivery account variable)
-- `GOVDELIVERY_PASSWORD` (GovDelivery account variable)
-- `SUBSCRIPTION_SUCCESS_URL` (Forwarding location on Subscription Success)
+**NOTE:** If you use ZSH you'll need to use
```

Comment:
I've tried to get in the practice putting notes in blockquotes to style them differently, e.g.:

```
> **NOTE:** If you use ZSH you'll need to use
   [zsh-autoenv](https://github.com/Tarrasch/zsh-autoenv), but we can't provide support for issues that may arise.
```